include/dlpack/dlpack.h

```diff
@@ -144,8 +149,8 @@ typedef struct {
    * \endcode
    */
   void* data;
-  /*! \brief The device context of the tensor */
-  DLContext ctx;
+  /*! \brief The device of the tensor */
+  DLDevice device;
```

Comment:
@tqchen @icemelon9 Unfortunately this breaks existing C code. Could you please revert the field name to `ctx`?

Reply:
@leofang this is intentional to keep the code consistent. The existing codebase can either stay at previous version, or upgrade the code to move to the new version. Exchange won't be affected by the version being used

Reply:
Ah OK, thanks. I failed to see this implication earlier.